### PR TITLE
Oracle price example documentation

### LINKED ISSUE #2615: Add doc for oracle signing BTC/USD price
basically this page except for a numeric outcome

https://bitcoin-s.org/docs/next/oracle/oracle-election-example

### PR DESCRIPTION
**Pull request by Christewart**
fixes #2615

### CODE REVIEW

File: docs/oracle/oracle-price-example.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+For this example, we will pick `numDigits=17`. This means we can sign a BTC/USD price
+in between `[0,2^17-1]` or `[$0-$131,071]`. If the BTC/USD price is outside of your
+predetermined interval, you need to sign the min (`0`) or the max (`$131,071`).
+
+## Setting up the BTC/USD oracle
+
+Given the parameters we specified above, we are ready to create our digit decompisition event!
+
+```
+ ./bitcoin-s-cli createdigitdecompevent "bitcoin-s-price-example" 1612396800 2 17 "BTC/USD" 0 --rpcport 12345
```
Comment (benthecarman) [inline docs/oracle/oracle-price-example.md]: I don't think we should include `--rpcport 12345` then it won't work if people are using the default setup

File: docs/oracle/oracle-price-example.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+1. The oracle announcement above (`fdd824fd02b9659...`)
+2. The timestamp that the event matures at (`1612396800`)
+
+Now you can schedule jobs to sign the event when the maturation time passes.
+
+## Signing the BTC/USD price for the oracle
+
+At the maturation time (`2020-02-04 00:00:00 UTC`) you need to check the BTC/USD price.
+
+For the purposes of this example, we are going to assume the BTC/USD price was `$42,069` BTC/USD.
```
Comment (benthecarman) [inline docs/oracle/oracle-price-example.md:87]: lol

**Comment by benthecarman** [comment]
Otherwise lgtm

**Review by nkohen — APPROVED** [review]
ACK

**Pull request merged**